dashboard_showcase.rb

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+def load_rails # TODO: extract to script
```

Comment:
Yeah I thought about this two, but adding dependencies between scripts is a whole new level of complexity so I prefer this approach for now.